src/cloud_db/cloud_db.py

```diff
@@ -117,19 +117,18 @@ def lookup_uid(self, uid: str):
         """
         logger.info(f'Looking up uid: {uid} in Supabase')
         result = None
-        order_type = None
+        nfc_type = None
         try:
             data = self.client.table("Order_Tag").select("*").eq("Tag_ID", uid).execute()
             if len(data.data) > 0:
                 order = self.client.table("Order_Tag_Group").select("Order_ID").eq("Group ID",data[0]["Order_Tag_Group_ID"]).execute()
```

Comment:
Additionally, you should handle when `order_tag[0]['order_tag_group_id']` is null, in which case you just make the `order_id = None`.